lib/core/presentation/widgets/collaborator/spotline_event_item.dart

```diff
@@ -0,0 +1,180 @@
+import 'package:app/theme/typo.dart';
+import 'package:cached_network_image/cached_network_image.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+
+class SpotlineEventItem extends StatelessWidget {
+  const SpotlineEventItem({
+    super.key,
+    required this.event,
+  });
+  final Event event;
+
+  DbFile? get eventPhoto => event.newNewPhotosExpanded?.isNotEmpty ?? false
+      ? event.newNewPhotosExpanded!.first
+      : null;
+
+  @override
+  Widget build(BuildContext context) {
+    final imageUrl = eventPhoto != null
+        ? ImageUtils.generateUrl(
+            file: eventPhoto,
+            imageConfig: ImageConfig.eventPhoto,
+          )
+        : '${AppConfig.assetPrefix}/assets/images/no_photo_event.png';
+    final colorScheme = Theme.of(context).colorScheme;
+    return Container(
+      width: 128.w,
+      decoration: ShapeDecoration(
+        shape: RoundedRectangleBorder(
+          borderRadius: BorderRadius.circular(
+            LemonRadius.small,
+          ),
+          side: BorderSide(
+            width: 1.w,
+            color: colorScheme.outline,
+          ),
+        ),
+      ),
+      child: Stack(
+        children: [
+          Positioned.fill(
+            child: ClipRRect(
+              borderRadius: BorderRadius.circular(
+                LemonRadius.small,
+              ),
+              child: CachedNetworkImage(
+                fit: BoxFit.cover,
+                imageUrl: imageUrl,
+                placeholder: (_, __) => ImagePlaceholder.defaultPlaceholder(),
+                errorWidget: (_, __, ___) =>
+                    ImagePlaceholder.defaultPlaceholder(),
+              ),
+            ),
+          ),
+          Positioned.fill(
+            child: DecoratedBox(
+              decoration: BoxDecoration(
+                gradient: RadialGradient(
+                  center: Alignment.bottomCenter,
+                  radius: 0.8,
+                  colors: [
+                    LemonColor.black.withOpacity(0.5),
+                    Colors.transparent,
+                  ],
+                ),
+              ),
+            ),
+          ),
+          Positioned(
+            top: Spacing.superExtraSmall,
+            right: Spacing.superExtraSmall,
+            child: _EventJoinCount(count: event.accepted?.length ?? 0),
+          ),
+          Positioned(
+            left: Spacing.xSmall,
+            bottom: Spacing.xSmall,
+            child: _EventInfo(event),
+          ),
+        ],
+      ),
+    );
+  }
+}
+
+class _EventInfo extends StatelessWidget {
+  const _EventInfo(this.event);
+  final Event event;
+
+  @override
+  Widget build(BuildContext context) {
+    final colorScheme = Theme.of(context).colorScheme;
+    return Column(
+      mainAxisSize: MainAxisSize.min,
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        SizedBox(
+          width: 100.w,
```

Comment:
dont need to set fixed width for the item, let use padding to narrow down the horizontal space of the widget